components/molecule/modal/src/index.js

```diff
@@ -242,6 +240,10 @@ MoleculeModal.propTypes = {
    * The callback function to be called when the animation ends, defined by the HoC WithAnimation
    */
   onAnimationEnd: PropTypes.func,
+  /**
+   * If true children won't be wrapped with content
```

Comment:
the explanation might not have ditails about the component implementation. What does content means? How does it affect (behavior-visually)?